core/chains/evm/txmgr/transmitchecker.go

```diff
@@ -168,9 +180,36 @@ func (v *VRFV1Checker) Check(
 		return nil
 	}
 
+	h, err := v.HeaderByNumber(ctx, nil)
+	if err != nil {
+		l.Errorw("Failed to fetch latest header. Attempting to transmit anyway.",
+			"err", err,
+			"ethTxID", tx.ID,
+			"meta", tx.Meta,
+		)
+		return nil
+	}
+
+	r, err := v.TransactionReceipt(ctx, *meta.RequestTxHash)
```

Comment:
I think we should check if `meta.RequestTxHash` is `nil` earlier in the func and `return nil` if it is indeed nil (i.e `Attempting to transmit anyway`)